### PR TITLE
Phrase changes, docker image change, and issue templates

### PR DESCRIPTION
**Pull request by justetz**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```
Comment (ddbruce) [inline .github/ISSUE_TEMPLATE/bug_report.md]: How come the R is capitalized but no other header has sentence case?

File: .github/ISSUE_TEMPLATE/support-question.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+title: ''
+labels: question
+assignees: ''
+
+---
+
+**Describe the question/request**
+A clear and concise description of what you need assistance with.
+
+**To Reproduce**
```
Comment (ddbruce) [inline .github/ISSUE_TEMPLATE/support-question.md]: Same here

**Comment by justetz** [comment]
As discussed, we will address wording of issue templates for all repos after this PR

**Comment by justetz** [comment]
Bump @ddbruce

**Pull request merged**